includes/helpers.inc

```diff
@@ -206,16 +143,6 @@ function _tide_import_single_config($config_name, array $locations = [], $overri
   try {
     $config_importer->import();
     \Drupal::cache('config')->delete($config_name);
-    // If configuration was set to override, then ensure it is written to active
-    // config.
-    if ($override) {
```

Comment:
Is this removed for the same reason that the delete function probably doesn't work?

Reply:
same, removed what were added by the previous implementation of SDPA-309.

Reply:
I wanted to point out that this code caused a problem with deployments. This is due to the fact that it was re-generating the same existing config with new UUIDs in different environment. So, the config was correctly exported in local, passed ci and develop env, but would not deploy to production since UUIDs would be re-generated as a part of the update, which runs _before_ config import.

Not sure what you are doing with this code right now - moving it to another place etc. - but there should be a flag to allow to skip writing of the config.

Reply:
If I understand it correctly, it is being removed completely. Writing of the config should never be done as it will never work outside of a local environment @alexdesignworks.